### PR TITLE
First pass, fixing spacing issues.

### PR DESCRIPTION
**Pull request by lkduerig**

### CODE REVIEW

File: public/components/people/people.js [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
 angular.module('rmtPeopleController', [])
 
-  .controller('PeopleController', function($rootScope, $scope, $http, Persons, Person) {
+  .controller('PeopleController', function ($rootScope, $scope, $http, Persons, Person) {
     $scope.formData = {};
-    if($rootScope.message) {
-      if($rootScope.message.section != 'people' || $rootScope.message.status == 'error' ){
+    if ($rootScope.message) {
+      if ($rootScope.message.section !== 'people' || $rootScope.message.status === 'error') {
         $rootScope.message = {}
```
Comment (amolnw2778) [inline public/components/people/people.js]: Missing `;` at the end. Not mandatory, good to have.

File: public/components/people/people.js [hunk 2]
```diff
@@ -15,17 +15,17 @@ angular.module('rmtPeopleController', [])
     // use service to get all persons
     
     Persons.get()
-    .success(function(data) {
+    .success(function (data) {
```
Comment (amolnw2778) [inline public/components/people/people.js]: Add indent here.

File: public/components/people/people.js [hunk 3]
```diff
@@ -15,17 +15,17 @@ angular.module('rmtPeopleController', [])
...
-          if(!angular.isUndefined(person.skills)){
+        _.each ($scope.persons, function (person) {
+          if (!angular.isUndefined(person.skills)) {
             person.skillsList = person.skills.toString();
           }
         });
     })
-    .error( function(){
+    .error(function () {
       console.log('error get')
```
Comment (amolnw2778) [inline public/components/people/people.js]: Please make messages public.

File: public/components/people/people.js [hunk 4]
```diff
@@ -35,20 +35,20 @@ angular.module('rmtPeopleController', [])
 
 // People form view
 
-  .controller('PeopleFormController', function($rootScope, $scope, $http, $stateParams, $location, Persons, Person, Skills) {
+  .controller('PeopleFormController', function ($rootScope, $scope, $http, $stateParams, $location, Persons, Person, Skills) {
     $scope.formData = {};
     $scope.id = $stateParams.id;
-    $scope.showUpdateButtons = $scope.id.length>0 ? true : false;
+    $scope.showUpdateButtons = $scope.id.length > 0 ? true : false;
```
Comment (amolnw2778) [inline public/components/people/people.js]: Did you tired `$scope.showUpdateButtons = $scope.id.length > 0;`? This should make it boolean.

File: public/components/people/people.js [hunk 5]
```diff
@@ -35,20 +35,20 @@ angular.module('rmtPeopleController', [])
 
 // People form view
 
-  .controller('PeopleFormController', function($rootScope, $scope, $http, $stateParams, $location, Persons, Person, Skills) {
+  .controller('PeopleFormController', function ($rootScope, $scope, $http, $stateParams, $location, Persons, Person, Skills) {
```
Comment (amolnw2778) [inline public/components/people/people.js]: `Persons, Person` is confusing to me. What's the difference in both?

File: public/components/people/people.js (hunk 2 above)
Comment (lkduerig) [inline public/components/people/people.js]: Ignore indentation on this PR, I will do that in the next PR.

File: public/components/people/people.js [hunk 6]
```diff
@@ -57,8 +57,8 @@ angular.module('rmtPeopleController', [])
 
     $scope.requiredFields = ['name', 'contact'];
 
-    $scope.validateForm = function(formData) {
-      _.each ($scope.requiredFields, function(field) {
+    $scope.validateForm = function (formData) {
+      _.each ($scope.requiredFields, function (field) {
```
Comment (amolnw2778) [inline public/components/people/people.js]: Remove space after `_.each`.

File: public/components/people/people.js (hunk 3 above)
Comment (lkduerig) [inline public/components/people/people.js]: Just code cleaning now. I don't know what this is doing here.

File: public/components/people/people.js (hunk 4 above)
Comment (lkduerig) [inline public/components/people/people.js]: Nope. sticking with formatting. I think you are right, but I don't want to mess with it in this commit.

Comment (lkduerig) [inline public/components/people/people.js]: I'll put a todo

File: public/components/people/people.js [hunk 7]
```diff
@@ -69,9 +69,9 @@ angular.module('rmtPeopleController', [])
         // GET ================================================
         // when landing on the page, get all persons and show them
         // use service to get all persons
-        if($scope.id.length > 0){
+        if ($scope.id.length > 0) {
```
Comment (amolnw2778) [inline public/components/people/people.js]: `$scope.id.length` is enough here.

File: public/components/people/people.js (hunk 4 above)
Comment (lkduerig) [inline public/components/people/people.js]: Nevermind, I changed it anyway.

File: public/components/people/people.js [hunk 8]
```diff
@@ -80,28 +80,28 @@ angular.module('rmtPeopleController', [])
...
             // validate the formData to make sure that something is there
             // if form is empty, nothing happens
             if (!$.isEmptyObject($scope.formData) && $scope.validateForm($scope.formData)) {
                 // call the create function from our service, returns a promise object
                 Persons.create($scope.formData)
                   // if creation is successful, call get function to get all the new stuff
-                .success(function(data) {
-                    $rootScope.message = { msg :'User ' + $scope.formData.name + ' added.', section : 'people', status : 'success'};
+                .success(function (data) {
+                    $rootScope.message = {msg :'User ' + $scope.formData.name + ' added.', section : 'people', status : 'success'};
```
Comment (amolnw2778) [inline public/components/people/people.js]: Missing space after colon `:'User`

File: public/components/people/people.js [hunk 9]
```diff
@@ -80,28 +80,28 @@ angular.module('rmtPeopleController', [])
...
                     $location.path('/api/people');
                 });
             }
           else {
-              $rootScope.message = { msg :'Please fill all required fields.', section : 'people', status : 'error'};
+              $rootScope.message = {msg :'Please fill all required fields.', section : 'people', status : 'error'};
```
Comment (amolnw2778) [inline public/components/people/people.js]: Missing space after colon.

File: public/components/people/people.js [hunk 10]
```diff
@@ -109,45 +109,45 @@ angular.module('rmtPeopleController', [])
...
             // validate the formData to make sure that something is there
             // if form is empty, nothing happens
             if (!$.isEmptyObject($scope.formData) && $scope.validateForm($scope.formData)) {
 
                 Person.put($scope.formData)
                 // if creation is successful, call get function to get all the new stuff
-                .success(function(data) {
-                    $rootScope.message = { msg :'User ' + $scope.formData.name + ' updated.', section : 'people', status : 'success'};
+                .success(function (data) {
+                    $rootScope.message = {msg :'User ' + $scope.formData.name + ' updated.', section : 'people', status : 'success'};
```
Comment (amolnw2778) [inline public/components/people/people.js]: Missing space after colon.

File: public/components/people/people.js [hunk 11]
```diff
@@ -109,45 +109,45 @@ angular.module('rmtPeopleController', [])
...
         // delete a person after checking it
-        $scope.deletePerson = function(id, name) {
+        $scope.deletePerson = function (id, name) {
 
             Persons.delete(id)
             // show remaining persons
-            .success(function(data) {
-                $rootScope.message = { msg :'User ' + name + ' deleted.', section : 'people', status : 'success'};
+            .success(function (data) {
+                $rootScope.message = {msg :'User ' + name + ' deleted.', section : 'people', status : 'success'};
```
Comment (amolnw2778) [inline public/components/people/people.js]: Missing space after colon.

File: public/components/people/people.js [hunk 12]
```diff
@@ -109,45 +109,45 @@ angular.module('rmtPeopleController', [])
...
                 $location.path('/api/people');
             });
         };
 
-        $scope.addSkill = function(name){
+        $scope.addSkill = function (name) {
           $scope.selectedskills.push({'name': name});
           $scope.formData.personskills = $scope.selectedskills;
```
Comment (amolnw2778) [inline public/components/people/people.js]: Remove empty lines.

File: public/components/people/people.js [hunk 13]
```diff
@@ -177,7 +177,7 @@ angular.module('rmtPeopleController', [])
 ])
 
 .directive('peopleForm',[
```
Comment (amolnw2778) [inline public/components/people/people.js]: Add space before square bracket. `[`

File: public/components/projects/projects.js [hunk 14]
```diff
@@ -3,10 +3,10 @@
 angular.module('rmtProjectController', [])
     
     //Project list
-    .controller('ProjectController', function( $rootScope, $scope, $http, Projects, Project, Persons, Person ) {
+    .controller('ProjectController', function ($rootScope, $scope, $http, Projects, Project, Persons, Person) {
         $scope.formData = {};
-        if($rootScope.message) {
-          if($rootScope.message.section != 'project' || $rootScope.message.status == 'error' ){
+        if ($rootScope.message) {
+          if ($rootScope.message.section !== 'project' || $rootScope.message.status === 'error'){
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Space before `{`

File: public/components/projects/projects.js [hunk 15]
```diff
@@ -15,28 +15,28 @@ angular.module('rmtProjectController', [])
...
         
         Projects.get()
-        .success(function(data) {
+        .success(function (data) {
             //$scope.skills = skills;
             $scope.projects = data;
 
         })
-        .error( function(){
+        .error(function (){
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Add space before `{`

File: public/components/projects/projects.js [hunk 16]
```diff
@@ -15,28 +15,28 @@ angular.module('rmtProjectController', [])
...
 
     })
     
     //Project form
-    .controller('ProjectAddController', function( $rootScope, $scope, $http, $stateParams, $location, Projects, Project, Persons, Person ) {
+    .controller('ProjectAddController', function ($rootScope, $scope, $http, $stateParams, $location, Projects, Project, Persons, Person) {
         $scope.formData = {};
         $scope.id = $stateParams.id;
-        $scope.showUpdateButtons = $scope.id.length>0 ? true : false;
+        $scope.showUpdateButtons = $scope.id.length > 0 ? true : false;
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: See above mentioned logic.

File: public/components/projects/projects.js [hunk 17]
```diff
@@ -15,28 +15,28 @@ angular.module('rmtProjectController', [])
...
         $rootScope.message = {}
 
         $scope.requiredFields = ['name', 'contact'];
 
-        $scope.validateForm = function(formData) {
-          _.each ($scope.requiredFields, function(field) {
+        $scope.validateForm = function (formData) {
+          _.each ($scope.requiredFields, function (field) {
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Remove space after each

File: public/components/projects/projects.js [hunk 18]
```diff
@@ -48,9 +48,9 @@ angular.module('rmtProjectController', [])
         // GET ================================================
         // when landing on the page, get all persons and show them
         // use service to get all persons
-        if($scope.id.length > 0){
+        if ($scope.id.length > 0){
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: See logic given above.

File: public/components/projects/projects.js [hunk 19]
```diff
@@ -118,7 +118,7 @@ angular.module('rmtProjectController', [])
 
   })
 .directive('projectsForm',[
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Space before `[`

File: public/components/projects/projects.js [hunk 20]
```diff
@@ -127,7 +127,7 @@ angular.module('rmtProjectController', [])
     }
 ])
 .directive('projectList',[
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Space before `[`

File: public/components/projects/projects.js [hunk 21]
```diff
@@ -136,14 +136,14 @@ angular.module('rmtProjectController', [])
...
 ])
 .directive('ngConfirmClick', [
-  function(){
+  function (){
     return {
       priority: -1,
       restrict: 'A',
-      link: function(scope, element, attrs){
-        element.bind('click', function(e){
+      link: function (scope, element, attrs){
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Space before `{`

File: public/components/projects/projects.js [hunk 22]
```diff
@@ -136,14 +136,14 @@ angular.module('rmtProjectController', [])
     }
 ])
 .directive('ngConfirmClick', [
-  function(){
+  function (){
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Space before `{`

File: public/components/projects/projects.js [hunk 23]
```diff
@@ -127,7 +127,7 @@ angular.module('rmtProjectController', [])
     }
 ])
 .directive('projectList',[
-    function(){
+    function (){
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Space before `{`

File: public/components/projects/projects.js [hunk 24]
```diff
@@ -136,14 +136,14 @@ angular.module('rmtProjectController', [])
...
     return {
       priority: -1,
       restrict: 'A',
-      link: function(scope, element, attrs){
-        element.bind('click', function(e){
+      link: function (scope, element, attrs){
+        element.bind('click', function (e){
           var message = attrs.ngConfirmClick;
-          if(message && !confirm(message)){
+          if (message && !confirm(message)){
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Space before `{`

File: public/components/projects/projects.js [hunk 25]
```diff
@@ -136,14 +136,14 @@ angular.module('rmtProjectController', [])
...
 .directive('ngConfirmClick', [
-  function(){
+  function (){
     return {
       priority: -1,
       restrict: 'A',
-      link: function(scope, element, attrs){
-        element.bind('click', function(e){
+      link: function (scope, element, attrs){
+        element.bind('click', function (e){
```
Comment (amolnw2778) [inline public/components/projects/projects.js]: Space before `{`

File: public/components/people/people.js (hunk 7 above)
Comment (lkduerig) [inline public/components/people/people.js]: Again, I was focusing on code formatting, not changing other people's code. If this breaks anything, I'm blaming you. :)

File: public/components/people/people.js (hunk 5 above)
Comment (lkduerig) [inline public/components/people/people.js]: Not sure, but one is singular and the other is plural.

Comment (amolnw2778) [inline public/components/people/people.js]: Still not clear. But let it for now.

**Pull request merged**